clang/test/Parser/cxx-variadic-func.cpp

```diff
@@ -6,3 +6,24 @@ void f(...) {
 }
 
 void h(int n..., int m); // expected-error {{expected ')'}} expected-note {{to match}}
+
+
+namespace GH153445 {
+void f(int = {}...);
+
+struct S {
+  void f(int = {}...);
```

Comment:
So `ConsumeAndStoreInitializer` is being called by `ParseCXXNonStaticMemberInitializer`, so I would assume this would not fix the static member functions. We should test this.